allure-generator/src/main/javascript/components/side-nav/styles.scss

```diff
@@ -44,7 +44,7 @@
 
   &__footer {
     margin-top: auto;
-    margin-bottom: $gap-size;
+    margin-bottom: $footer-gap-size;
```

Comment:
> When I added Hebrew and opened the language bar, the Hebrew language was sliced by the page (overflow).

@just-boris is there a way to fix it in simple way? I guess we can change position from `right` to `top-right`

![screen shot 2017-08-20 at 20 32 12](https://user-images.githubusercontent.com/2149631/29496952-b82704bc-85e6-11e7-9f47-174e01562479.png)